src/views/studentcompleteregistration/studentcompleteregistration.jsx

```diff
@@ -86,11 +86,18 @@ var StudentCompleteRegistration = intl.injectIntl(React.createClass({
             method: 'post',
             useCsrf: true,
             formData: submittedData
-        }, function (err, body) {
+        }, function (err, body, res) {
             this.setState({waiting: false});
             if (err) return this.setState({registrationError: err});
             if (body.success) return this.advanceStep(formData);
-            this.setState({registrationErrors: body.errors});
+            this.setState({
+                registrationErrors:
```

Comment:
That response returns an object from the Django form errors, so there could be multiple errors.  This view is set up to output the form error object rather than one thing.